deployment/E_heroku.md

```diff
@@ -1 +1,277 @@
-## FPO
+### What we'll need
+
+- a working application
+
+### Goals
+
+Our main goal for this is guide is to get a Phoenix application running on Heroku.
+
+## Steps
+
+- Sign up for Heroku and install the Toolbelt
+- Create the Heroku application
+- Add the Phoenix static buildpack
+- Make our project Heroku-ready
+- Deploy!
+
+## Sign up for Heroku and install the Toolbelt
+
+Signing up to Heroku is very simple, just head over to [https://signup.heroku.com/www-header](https://signup.heroku.com/www-header).
+
+Once this is done download the Heroku Toolbert installer for your platform at [https://toolbelt.heroku.com/](https://toolbelt.heroku.com/).
+
+This will allow you to access Heroku commands from your shell.
+
+## Creating the Heroku application
+
+Now that we have the Toolbelt installed, let's create the Heroku application. In your project directory, run:
+
+```console
+$ heroku create --buildpack "https://github.com/HashNuke/heroku-buildpack-elixir.git"
+Creating mysterious-meadow-6277... done, stack is cedar-14
+Buildpack set. Next release on mysterious-meadow-6277 will use https://github.com/HashNuke/heroku-buildpack-elixir.git.
+https://mysterious-meadow-6277.herokuapp.com/ | https://git.heroku.com/mysterious-meadow-6277.git
+Git remote heroku added
+```
+
+The option we are passing allows us to specify the [Elixir buildpack](https://github.com/HashNuke/heroku-buildpack-elixir) we want Heroku to use.
+A [buildpack](https://devcenter.heroku.com/articles/buildpacks) is a collection of scripts that Heroku uses to prepare your code for execution.
+
+## Adding the Phoenix static buildpack
+
+In order to successfully deploy a Phoenix app, static assets need to be compiled. Let's add the [Phoenix static buildpack](https://github.com/gjaldon/heroku-buildpack-phoenix-static) that is going to take care of that for us.
```

Comment:
Let's see if we can eliminate the passive voice here. What do you think about this?

We need to compile static assets for a successful Phoenix deployment. The [Phoenix static buildpack](https://github.com/gjaldon/heroku-buildpack-phoenix-static) can take care of that for us, so let's add it now.